conf/application.conf

```diff
@@ -76,6 +79,15 @@ microservice {
   }
 }
 
+mongodb {
+  uri = "mongodb://localhost:27017/income-tax-penalties-appeals-frontend"
+  ttl = 15 minutes
+  encryption {
+    enabled = false
+    key = ""
+  }
```

Comment:
Actually, we can just delete this for now, because we aren't using this config yet. If we need to encrypt the answers then we can add this back in when needed.